README.md

```diff
@@ -56,12 +56,15 @@ To verify the native `.node` packages, follow the same steps as above using `mon
 
 Only the following version combinations with the [MongoDB Node.js Driver](https://github.com/mongodb/node-mongodb-native) are considered stable.
 
-|               | `@mongodb-js/zstd@1.x` |
-| ------------- | ---------------------- |
-| `mongodb@6.x` | ✓ `^1.1.0`           |
-| `mongodb@5.x` | ✓                      |
-| `mongodb@4.x` | ✓                      |
-| `mongodb@3.x` | N/A                    |
+|                  | `@mongodb-js/zstd@1.x` | `@mongodb-js/zstd@2.x` |
+| ---------------- | ---------------------- | ---------------------- |
+| `mongodb@>=6.11` | ✓ `^1.1.0`             | `^2.0.0`               |
+| `mongodb@<6.11`  | ✓ `^1.1.0`             | N/A                    |
+| `mongodb@5.x`    | ✓                      | N/A                    |
+| `mongodb@4.x`    | ✓                      | N/A                    |
+| `mongodb@3.x`    | N/A                    | N/A                    |
+
+`@mongodb-js/zstd@2.x` will work with `mongodb@<6.11` but will produce peer dependency errors upon installation.
```

Comment:
I understand the sentiment but if there were issues with compatibility I don't think we would go back to patch it right?

I would omit this. If you feel like it is worth mentioning it then can we attach a disclaimer of sorts that it's untested/unsupported?

Reply:
It's not listed in the compatibility table.  Does that suffice in terms of "disclaimer / unsupported"?

Reply:
No I think it should be explicit _if_ we're adding a sentence that says "it will work".  

It is not clear if the reason it is not in the table is _only_ because of the peer dep error or because it is not supported. I want to draw a line between 'works' and 'will be fixed if it does not'.